<?xml version="1.0"?>
<launch>
   <!-- 颜色检测模式 -->
    <node name="yolo11_inference_color" pkg="perception" type="yolo11_inference.py" output="screen">
        <param name="detection_method" value="yolo"/>
        <param name="input_topic" value="/standard_vtol_0/camera/image_raw"/>
        <param name="output_topic" value="/yolo11/detection_image"/>
        
        <!-- 颜色检测参数 -->
        <param name="min_contour_area" value="500"/>
        <param name="morphology_kernel_size" value="5"/>
        <param name="gaussian_blur_size" value="5"/>
        
        <!-- 点云发布设置 -->
        <param name="publish_pointcloud" value="true"/>
        <param name="pointcloud_frame_id" value="map"/>
        <param name="max_points_per_cloud" value="1000"/>
    </node>
</launch>
